MANIFEST.in

```diff
@@ -1,3 +1,3 @@
 include LICENSE
 include README.md
-recursive-include major_event_log *.html *.xml *.css *.js *.jpg *.jpeg *.png *.gif
+recursive-include major_event_log/templates *
```

Comment:
There shouldn't be any python files in there at all. Those are only going to be .html or .xml files.